### PR TITLE
doc: update reflection-milestone2.md

### PR DESCRIPTION
**Pull request by jennifer-hoang**
Hello all,

Here is my draft for our reflection. Please let me know what you think, and if you have any additional thoughts you would like to add.

Thanks!

### CODE REVIEW

**Review by michelle-wms — APPROVED** [review]
looks good to me!

**Review by christopheralex — APPROVED** [review]
Looks good :) Great work Jennifer

File: doc/reflection-milestone2.md
Comment (xiangwxt) [inline doc/reflection-milestone2.md:22]: Maybe we can improve these within the milestone 2

File: doc/reflection-milestone2.md
Comment (xiangwxt) [inline doc/reflection-milestone2.md:15]: This suffices the need of our target users, content managers at Spotify, to get music tracks insights visually and quickly to support their data driven decision making process.

**Review by xiangwxt — APPROVED** [review]
Hi Jennifer, the reflection is quite comprehensive of what we have done, good job. I just have a few comments and suggestions. Hope those would be helpful.

**Comment by christopheralex** [comment]
I am sorry I merged this PR . Would wait for everyone's approval next time.
I can make a github issue of this for Jenn and ourselves to refer.
Thanks you for you review Thea Wenxin

**Pull request merged**
